### PR TITLE
resource.fonts.copacetic 1.0.0

### PR DESCRIPTION
**Pull request by realcopacetic**
Initial repo submission for resource.fonts.copacetic

### Description
A new resource dependency for the skin Copacetic, submitted to rep-skins repo. This resource provides additional font support, a typeset combining Inter for latin characters with Noto Sans for CJK.

### Checklist:
- [X] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [X] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-resources/blob/master/CONTRIBUTING.md) document
- [X] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practice but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-plugins/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.

### CODE REVIEW

**Review by romanvm — APPROVED** [review]

**Review by romanvm — CHANGES_REQUESTED** [review]
Please fix errors found by the addon checker

**Review by pkscout — APPROVED** [review]

**Comment by realcopacetic** [comment]
hello @romanvm apologies for the errors. I think I have fixed them all. Regards,
Ash

**Comment by realcopacetic** [comment]
Sorry @romanvm can you shed any light on these 2 errors as I don't understand them?

> ERROR: kodi.resource.font is not a valid extension point

I thought this was a valid extension point as listed here: https://kodi.wiki/view/Addon.xml
Have I used it incorrectly?

> ERROR: Schema validation failed for root addon element

I'm not sure where I've gone wrong with the schema. Is this related to the error above?

**Comment by romanvm** [comment]
I'll check this. The extension point should be valid.

**Comment by realcopacetic** [comment]
> I'll check this. The extension point should be valid.

Thanks!

**Comment by romanvm** [comment]
Please also fix the fanart size.

**Comment by realcopacetic** [comment]
> Please also fix the fanart size.

Done and squashed commits

**Comment by romanvm** [comment]
LGTM but addon checker needs to be fixed.

**Review by romanvm — APPROVED** [review]

**Comment by realcopacetic** [comment]
Thanks @romanvm do you know the expected time frame for that? I'm wondering whether it's better to remove this as a dependency from my skin so it will at least work for most people in the official repo without added CJK language support.

**Comment by romanvm** [comment]
I cannot really tell when the checker will be fixed. So if this font package is not critical for your skin it's better to skip it for now. I'd merge it if I could but this repo is configured in such way that green checker run is mandatory.

**Comment by enen92** [comment]
@realcopacetic I took some time to update addon-checker and adding support for font addons in the PR above.
Your addon xml is wrong though:

```
INFO: This is a new addon
failed validating {'point': 'kodi.resource.font', 'compile': 'false'} with XsdAttributeGroup(['point']):

Reason: 'compile' attribute not allowed for element

Schema:

  <xs:complexType xmlns:xs="http://www.w3.org/2001/XMLSchema">
    <xs:attribute name="point" type="simpleIdentifier" use="required" />
  </xs:complexType>

Instance:

  <extension point="kodi.resource.font" compile="false" />
```

Please drop the `compile` attribute from the extension element, this is not valid according to the Kodi core code.

File: resource.fonts.copacetic/addon.xml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="resource.fonts.copacetic" name="Copacetic Extra Fonts" version="1.0.2" provider-name="realcopacetic">
+    <requires>
+        <import addon="kodi.resource" version="1.0.0" />
+    </requires>
+
+    <extension point="kodi.resource.font" compile="false" />
```
Comment (enen92) [inline resource.fonts.copacetic/addon.xml]:
```suggestion
    <extension point="kodi.resource.font"/>
```

**Review by enen92 — CHANGES_REQUESTED** [review]
After the addon.xml is fixed it can be merged

**Comment by realcopacetic** [comment]
> After the addon.xml is fixed it can be merged

Ah great thanks @enen92 - I think I copied that compile bit from a different resource addon I found as I couldn't find much documentation, thanks for fixing. That's now updated.

**Review by enen92 — APPROVED** [review]

**Comment by enen92** [comment]
Great. If you find some issue with addon-checker feel free to ping me directly, I try to maintain it overtime (best effort ofc)

**Pull request merged**